.travis.yml

```diff
@@ -8,10 +8,34 @@ go:
 services:
   - docker
 
+env:
+  global:
+    - COMMIT=${TRAVIS_COMMIT::8}
+    # DOCKER_USER
+    - secure: TNvbr5/d7raSovEtttxdrZl8tP7vCCzL56gKKhr2wF4ET5/iRbcfSP9zoFPEOlIdgmCHZhGTh+fED1Eqgyswv6HPOAfEuov5vtzNB9fkcI46/nRk7KbiDlcEuE2IFtwkijFDz6YdJlbPCozHa81/Ih6G20H61tgv6f0AsGGT9MR7DQ71cCu8xZykNDjEKTo7RF6GiqG2VYa+S1P3vCOKRv31ouo/a5SPP+1AIvAg8u++qWVC8WJixmkXnw2OplvCFgHS0dlT3FvUPjYtUtLens5gpBDo7kn7+Ba27m2D0IzkzDPW5sK0YMMZW61LLn8GLPiJtLqzUHovaJ2NcFfi07RQ4GSMnwnjP0nLQbgd0CzM2zJGJRcOTkYe7IEDrdcTBcljZdZAJdoJEzrGYYWRQcX0Kyjc83ghX3A5s+CQWlPElQrkBB7KhNZ+w2Cn4+Mr6zOiRnBYg1NIUV2eGHMNnC4HI9RqgvA1QqcT5YHWqpz20sddHx1kgzgh8vOW8csiFon/Wrvyb2TaemzsKxIlT/UZZfDuyWG/Lvm4oxmTp1GrgQsC2iJjox4z6VIxbhykZEqNU1dhY6KuvgjEGetxk2j/NVfI8Qb4tvWqKoXq5Buap/J0AWjxWjGbrIGZbz5FgzfEP33WR8X2Oh5Cy+TMl1v0+YBAB3OaMpe/Qe2rGlk=  # DOCKER_PASS
+    # DOCKER_PASS
+    - secure: mX3/Gwv5esCjU9GfqDaXbTslA0UOY9q45+rnW58X6z1tY2pFOts5uhtLve3YneGn3IN0n+m5mcljim72vKNQaPJCSodYfcfrqth2YqkYTXNWykOIdlLs1u58pYYGRXJaAlD4EgQysB/nAaQYO00/n2tBq4vEmu0dikbJer97smStkokLBiz42tJbvY6mikqnHHbRAA4dw5MHwYikDkI82oKxUsB8horetMmXgwTkT3AiE5UufXm3iYxpZ48KTYBW8WbQSQV1T1nriisZVErbYh69QLaLrEK3/WYiN3r4fOag4JvMA5GuPpuVGQfKzXUuFptG54VBbPKCgmcXA/DlaEQ6HILsfHdl29/AXHa59dVjWGfQ5f/N4VWaIVXhE0qWo+/WIoyr7SaLybyGVe3gf1vDaDFlfVlrE7j13H+XKiwzfOeaqnTrcNc/MSCFCtnUbjvX4UlXCbE4gVbCuEsjAPQ8mI/MMaCWLxhaYTW/cSyGRTGMLwe4Q8uAPd3BQat63byLo3BfK+icyw24RbSGBDRu5FZORK164MMjr0+UKVT3gh5/JdFxLkdOGjJL/9QuppNShI5GBIctIg15AQJNkhSLm8DxUTNHhjjBgXzqNhDVmUQLQVgqCWlHh1HVF5eiIQXBNGBE1Lw9v9dGTUHlPUUqeJKsxEyYWfZuY4tNhzU=
+
 install:
+  - nvm version
+  - node -v
   - make install_ci
 
 script:
   - make test_ci
   - travis_retry goveralls -coverprofile=cover.out -service=travis-ci || true
 
+after_success:
+  - git clone https://github.com/uber/jaeger-ui
+  - cd jaeger-ui && npm install && npm run build && cd ..
```

Comment:
actually, why don't we have a make target for this? it will be needed when building / testing query service locally anyway, less noise in travis file.